examples/templates/configuration/raw-keyrings/raw-rsa-keyring-encrypt-with-public-only.md

```diff
@@ -0,0 +1,188 @@
+[//]: # "Copyright Amazon.com Inc. or its affiliates. All Rights Reserved."
+[//]: # "SPDX-License-Identifier: CC-BY-SA-4.0"
+
+# Raw RSA keyring example using separate keyrings for encrypt and decrypt
+
+Implementations of this example MUST follow the rules defined in
+[Example Templates](../../../examples.md#example-templates).
+
+## Implementations
+
+- [Python (DEV)](https://github.com/aws/aws-encryption-sdk-python/blob/keyring/examples/src/keyring/raw_rsa/public_private_key_separate.py)
+- [Java (DEV)](https://github.com/aws/aws-encryption-sdk-java/blob/keyring/src/examples/java/com/amazonaws/crypto/examples/keyring/rawrsa/PublicPrivateKeySeparate.java)
+
+## Definitions
+
+### Conventions used in this document
+
+The key words
+"MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT",
+"SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL"
+in this document are to be interpreted as described in
+[RFC 2119](https://tools.ietf.org/html/rfc2119).
+
+## Header
+
+```python
+# One of the benefits of asymmetric encryption
+# is that you can encrypt with just the public key.
+# This means that you can give someone the ability to encrypt
+# without giving them the ability to decrypt.
+#
+# The raw RSA keyring supports encrypt-only operations
+# when it only has access to a public key.
+#
+# This example shows how to construct and use the raw RSA keyring
+# to encrypt with only the public key and decrypt with the private key.
+#
+# If your RSA key is in PEM or DER format,
+# see the ``keyring/raw_rsa/private_key_only_from_pem`` example.
```

Comment:
```suggestion
# see the `keyring/raw_rsa/private_key_only_from_pem` example.
```

np: The current example doesn't render properly. I'm not sure if my version will help at all, but it gets rid of the exta `'s being show

Reply:
That falls under "adjust for proper comment formatting.